src/components/expand-viewer/expand-viewer.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { ITheme } from '../../theme/types';
+
+export const expandViewButtonStyle = (theme: ITheme, isHovered: boolean) => ({
+  padding: '4px 12px',
+  borderRadius: '50px',
+  borderWidth: '1px',
+  borderStyle: 'solid',
+  /**
+   * since it's not possible to use :hover selector in inline React style I had to handle it manually,
+   * there could be added a css style which avoided since there is no css files in the project
+   */
+  backgroundColor: isHovered ? theme.colors.white : theme.colors.primaryShade2,
+  borderColor: theme.colors.primary,
```

Comment:
you don't need to access `theme` programmatically
since we use `emotion-theming`, you can just but `borderColor: 'primary'`. This is the case for every color usage

Reply:
same reason to avoid using styling from emotion CSS. gonna update to use rebass instead.